src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -228,9 +233,15 @@ export class QuickBar extends LitElement {
   }
 
   private _renderItem(item: QuickBarItem, index?: number) {
-    return isCommandItem(item)
-      ? this._renderCommandItem(item, index)
-      : this._renderEntityItem(item, index);
+    if (isCommandItem(item)) {
+      return this._renderCommandItem(item, index);
+    }
+
+    if (isEntityItem(item)) {
```

Comment:
This is just... we check if it is not a command item, we already know that, because we checked that above...?

Reply:
`_renderEntityItem(item)` expects an `EntityItem`, but `item` is a `QuickBarItem` here. This would be okay except that the `EntityItem` interface now has the `altText` property.

So I'm no longer allowed to blindly send a `QuickBarItem` to the `_renderEntityItem` method due to mismatching types.
<img src="https://user-images.githubusercontent.com/1480827/114604953-6b820f80-9c4e-11eb-8a14-2a08d07d8c18.png" width=50% />

Reply:
`as EntityItem` ?

Reply:
Could do, but I try to avoid `X as Y` as it [weakens type safety](https://medium.com/swlh/why-you-should-avoid-type-assertions-in-typescript-5494e3d04dd).

Reply:
yes, but I rather do that than to add unneeded runtime checks 😄

Reply:
Okie doke

Reply:
Fixed